CONTRIBUTING.md

```diff
@@ -52,6 +56,7 @@ ensure that your code complies with some standard style conventions.
   ```
   - Make sure that the tests pass
   ```
+  python setup.py install
   python setup.py test
```

Comment:
maybe there is also a designated place in `setup.py` for test-only requirements?